docs/UserGuide.md

```diff
@@ -16,141 +16,254 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
    > Having any Java 8 version is not enough. <br>
    > This app will not work with earlier versions of Java 8.
 
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
+1. Download the latest `taskmanager.jar` from the [releases](../../../releases) tab.
 2. Copy the file to the folder you want to use as the home folder for your Address Book.
 3. Double-click the file to start the app. The GUI should appear in a few seconds.
    > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+4. Refer to the [Features](#features) section below for details of each feature available.<br>
 
 
 ## 2. Features
 
 > **Command Format**
->
-> * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
-> * Items with `...` after them can have multiple instances.
-> * Parameters can be in any order.
+> `Square brackets [ ]` denote a required field
+> `Curved brackets ( )` denote an optional field'
+> `...` denotes that you can have multiple instances
 
-### 2.1. Viewing help : `help`
+## 2.01 View help: `help`
+View all the available commands and how to use them
 
-Format: `help`
+_Format:_
+`help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+## 2.02 Add task: `add`
+Adds a task to your todo list
 
-### 2.2. Adding a person: `add`
+_Format:_
+`add [Name-of-Task] [Deadline] (#Tag-1) (#Tag-2) …`
```

Comment:
would be good if you guys can think about how spaces within "Name-of-Task" will work?